pkg/scheduler/core/equivalence/eqivalence.go

```diff
@@ -111,90 +145,84 @@ func (ec *EquivalenceCache) updateResult(
 		return
 	}
 	nodeName := nodeInfo.Node().GetName()
-	if _, exist := ec.algorithmCache[nodeName]; !exist {
-		ec.algorithmCache[nodeName] = AlgorithmCache{}
+	if _, exist := c.cache[nodeName]; !exist {
+		c.cache[nodeName] = make(predicateMap)
 	}
-	predicateItem := HostPredicate{
+	predicateItem := predicateResult{
 		Fit:         fit,
 		FailReasons: reasons,
 	}
 	// if cached predicate map already exists, just update the predicate by key
-	if predicateMap, ok := ec.algorithmCache[nodeName][predicateKey]; ok {
+	if predicates, ok := c.cache[nodeName][predicateKey]; ok {
 		// maps in golang are references, no need to add them back
-		predicateMap[equivalenceHash] = predicateItem
+		predicates[equivalenceHash] = predicateItem
 	} else {
-		ec.algorithmCache[nodeName][predicateKey] =
-			PredicateMap{
+		c.cache[nodeName][predicateKey] =
+			resultMap{
 				equivalenceHash: predicateItem,
 			}
 	}
 	glog.V(5).Infof("Updated cached predicate: %v for pod: %v on node: %s, with item %v", predicateKey, podName, nodeName, predicateItem)
 }
 
-// lookupResult returns cached predicate results:
-// 1. if pod fit
-// 2. reasons if pod did not fit
-// 3. if cache item is not found
-func (ec *EquivalenceCache) lookupResult(
+// lookupResult returns cached predicate results and a bool saying whether a
+// cache entry was found.
+func (c *Cache) lookupResult(
 	podName, nodeName, predicateKey string,
 	equivalenceHash uint64,
-) (bool, []algorithm.PredicateFailureReason, bool) {
-	ec.mu.RLock()
-	defer ec.mu.RUnlock()
+) (value predicateResult, ok bool) {
+	c.mu.RLock()
+	defer c.mu.RUnlock()
 	glog.V(5).Infof("Begin to calculate predicate: %v for pod: %s on node: %s based on equivalence cache",
```

Comment:
Now that you are at it, please replace the log message with:
"looking up equivalence cache for predicate: %v and pod: %s on node: %s"

Reply:
Done